release-pipeline.yml

```diff
@@ -1578,6 +1580,7 @@ stages:
           - script: |
               sudo apt-get install -y zip
               cp $(Build.ArtifactStagingDirectory)/azCopy-linux-signed/azcopy-*x86_64.deb .
+              rm -rf /usr/local/bin/azcopy
```

Comment:
shall we delete /usr/bin/azcopy as well here.

Reply:
sure will add it at both places